products/appengine/terraform.yaml

```
@@ -24,7 +24,36 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           project_id: "test-project"
         test_env_vars:
           org_id: :ORG_ID
-
+  StandardAppVersion: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: "apps/{{project}}/services/{{service}}/versions/{{version_id}}"
+    import_format: ["apps/{{project}}/services/{{service}}/versions/{{version_id}}"]
+    parameters:
```

```suggestion
    mutex: "apps/{{project}}/services/{{service}}"
    parameters:
```

Only 1 operation can happen on a service at a time, so we'll need to guard our service with a mutex.

---

Thank you @rileykarson.

---

Should I grant you access to my fork?